meshio/_mesh.py

```diff
@@ -10,6 +10,7 @@ def __init__(
         cell_data=None,
         field_data=None,
         node_sets=None,
+        element_sets=None,
```

Comment:
@nschloe Why isn't `node_sets={}` used here instead of `None`.  I'm not a great python programmer.

Reply:
Mutable default arguments are a no-no in Python; essentially there's only one of them that gets shared by all calls to the function. For the classic explanation see:

https://effbot.org/zone/default-values.htm

Reply:
Ha the more you know - thanks!